validation/governed-by-optimism_test.go

```diff
@@ -0,0 +1,23 @@
+package validation
+
+import (
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum-optimism/superchain-registry/validation/standard"
+
+	// "github.com/ethereum/go-ethereum/common"
+
+	"github.com/stretchr/testify/require"
+)
+
+func testGovernedByOptimism(t *testing.T, chain *ChainConfig) {
+	chainID := chain.ChainID
+	superchain := OPChains[chainID].Superchain
+
+	optimismMultisig := standard.Config.MultisigRoles[superchain].KeyHandover.L1.Universal["ProxyAdmin"]["owner()"]
+
+	if chain.GovernedByOptimism == true {
+		require.Equal(t, chain.Addresses.ProxyAdmin.String(), optimismMultisig, "Chains using Optimism governance must have their ProxyAdmin set to the Optimism multisig")
```

Comment:
Possibly related to my previous comment, you currently have a test failure here https://app.circleci.com/pipelines/github/ethereum-optimism/superchain-registry/10103/workflows/6ad5f5b4-8e89-4d1e-868d-14043b8fbc96/jobs/45810 suggesting that OPM is not governed by optimism :-) 

The existing test defined here https://github.com/ethereum-optimism/superchain-registry/blob/c56b99231b9033d4db7d64f4fe4eeca72d733a8a/validation/standard/standard-config-roles-mainnet.toml#L9-L11 is passing when I run `just validate 10`.